packages/core/src/lib/disk.ts

```diff
@@ -1,24 +1,42 @@
+import { debuglog } from 'node:util'
 import { Readable } from 'stream'
 
+import { HeadBucketCommand, S3Client } from '@aws-sdk/client-s3'
 import { Result } from '@latitude-data/core/lib/Result'
 import { env } from '@latitude-data/env'
 import { Disk, errors } from 'flydrive'
 import { FSDriver } from 'flydrive/drivers/fs'
 import { S3Driver } from 'flydrive/drivers/s3'
 import { WriteOptions } from 'flydrive/types'
 
+var debug_default = debuglog('flydrive:s3')
```

Comment:
In this case I tag these logs (temp hack to debug) as FlyDrive S3

Reply:
var? are we back to 2006 javascript? :trollface:

Reply:
i think carlos hasn't seen a var in real life, too young